CCPP v6.0.0 SciDoc updates (#919)
NCAR/ccpp-physics: physics/docs/pdftxt/GFS_SFCLYR.txt

The pertinent features of the GFS stable surface layer parameterization scheme are described in the appendix of Zheng et al. (2017) 
\cite zheng_et_al_2017.

\section version_gfsslay_enh CCPP Physics Updates
\version CCPP v6.0.0
\author Jongil Han (NCEP Environmental Modeling Center): Jongil.Han@noaa.gov

> - New canopy heat storage parameterization is implemented, which is achieved by reducing sensible heat flux into the PBL scheme as a function of surface roughness and vegetation fraction and helps to reduce nighttime cold and daytime warm 2-meter temperature biases over forested regions.

Reviewer: ```suggestion
- A new canopy heat storage algorithm was implemented. The reduction of the sensible heat flux into the PBL, as a function of surface roughness and vegetation fraction, helps to reduce nighttime cold and daytime warm 2-meter temperature biases over forested regions.
```

> - A sea spray effect parameterization is included to enhance sensible and latent heat fluxes, especially for strong wind conditions.

Reviewer: ```suggestion
- A sea spray effect algorithm was included to enhance sensible and latent heat fluxes, especially for strong wind conditions.
```

> - To better represent sub-grid scale turbulence variability in the surface layer, a new parameterization for maximum surface layer stability parameter is developed as an inverse function of the background turbulent eddy diffusivity.

Reviewer: ```suggestion
- To better represent sub-grid scale turbulence variability in the surface layer, a new algorithm for maximum surface layer stability parameter was developed as an inverse function of the background turbulent eddy diffusivity.
```

> - Thermal roughness length calculation over land has been modified.

Reviewer: ```suggestion
- The thermal roughness length calculation over land has been modified.
```

Author: Done
